client/providers/CallProvider/hooks/useVoipClient.ts

```diff
@@ -32,18 +32,17 @@ export const useVoipClient = (): UseVoipClientResult => {
 	const voipEnabled = useSetting('VoIP_Enabled');
 	const registrationInfo = useEndpoint('GET', 'connector.extension.getRegistrationInfoByUserId');
 	const membership = useEndpoint('GET', 'voip/queues.getMembershipSubscription');
-	const user = useUser();
+	const userId = useUserId();
```

Comment:
Hey 👀 not sure of a different approach here, but, I did use "user.extension" prop on other fix to prevent this call from happening if the user doesn't have an extension associated.

But, that breaks the fix 🤔 since I was using the "useUser" hook, unless we have a "useUserExtension" hook somewhere (or we create one)

Reply:
@KevLehman Can you please point me to the task? There are few of such kind which are dependent.

Reply:
As @KevLehman pointed out, the fix was breaking https://github.com/RocketChat/Rocket.Chat/pull/24752. Next commit fixes this.